packages/ui-components/src/components/Chart/Table/useChartDataExport.js

```diff
@@ -3,8 +3,8 @@
  *  Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
  */
 
-import { useDataTableExport } from '@tupaia/ui-components';
```

Comment:
@tcaiger this was throwing an error, is there any issue with importing the file directly?

Reply:
Yes, we can't import the file directly because the Chart (and Map) directory gets bundled separately to the rest of the ui-components. It's definitely a trip hazard the way it's set up at the moment.

Reply:
Right. What kind of issues does it cause? The way I've set it up here works in lesmis and tupaia.org, and the size of the file is still nice and small (e.g. `lib/map/index.js` is 55 KB after the change), so I'm not sure where to look for what might need fixing tbh!

Reply:
Just checked and it was 54 KB before, so seems like a pretty acceptable increase 😀

Reply:
Oh I mi-understood your message. I thought it was throwing an error after the change. The only issue that it causes is that it breaks the encapsulation between the 2 bundles which may be needed if we split them into different packages. However since that's just theoretical at this point, it is all good to pull in the file directly.

Reply:
Ah right! True, I see your point. Does fix an eslint issue and avoids confusion for now though, so let's split it properly if needed later. Thanks!